### DISCUSSION TITLE
Option to use shortened album name

### DISCUSSION THREAD

**Discussion by RogueOneEcho** [RogueOneEcho/caesura#113] [Ideas]
The logic to suggest shortened album names already exists but there is no option to actually use this value:

https://github.com/RogueOneEcho/caesura/blob/e803c44117541f4a2cc312b98bed54eaba582d7f/src/naming/shortener.rs#L39-L62

- [ ] Implement as `--shorten-directory-name`
- [ ] For consistency `verify` `spectrogram` `transcode` and `upload`  commands would need to accept this.
- [ ] Apply to `spectrogram` output dir
- [ ] Apply to `transcode` output dir
- [ ] Apply to `upload` `.torrent` files
- [ ] Batch command should verify that the option is NOT enabled otherwise it could cause issues with matches. 

Batch command will either skip the non-shortened version because it's too long. Or will skip because all formats are already uploaded.

Related: https://github.com/RogueOneEcho/caesura/discussions/58

**Comment by Asparghus** [comment]
That information could be save on the cache file for later commands. E.g. You run verify, spectrogram and transcode with --shorten-directory-name, but later when run upload forgot to use --shorten-directory-name.

**Comment by RogueOneEcho** [comment]
- [ ] If album title includes `(remix)` or similar then this may get cut by `--shorten-directory-name` which should be avoided

**Reply by RogueOneEcho** [reply]
For this reason I don't think there's value in shortening the album name and will close this as wontfix
